src/utils/tests.js

```diff
@@ -0,0 +1,6 @@
+import { ThemeProvider } from 'styled-components';
+import theme from '../theme/light.json';
+
+export const themedComponent = children => (
+  <ThemeProvider theme={theme}> {children} </ThemeProvider>
+);

```

Comment:
Maybe something like`renderWithTheme` will save us some extra lines in the future. 
```
import { render } from '@testing-library/react';

export const renderWithTheme = children => {
  return render(<ThemeProvider theme={theme}> {children} </ThemeProvider>);
};

Reply:
thanks!~